### PR TITLE
Issues documentation and configurable thresholds

### PR DESCRIPTION
**Pull request by HustonMmmavr**
I didn't forget about

- [x] Tests
- [x] Changelog
- [x] Documentation

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -18,6 +18,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 - Add new field GraphQL field `cluster{ issues {topic} }` and extend
   list of inspected topics with `replication`, `failover`, `memory`, `clock`.
+  Thresholds for issues with topic `memory` and `clock` are configurable with
+  `argparse` options. Set environment variable `TARANTOOL_DESYNC_THRESHOLD_WARNING`
+  (in seconds) to configure threshold for issues with topic `clock`. Set environment variable
+  `TARANTOOL_FRAGMENTATION_THRESHOLD_CRITICAL` and `TARANTOOL_FRAGMENTATION_THRESHOLD_WARNING`
+  (`number` in range [0, 1]) to configure thresholds for issues with topic `memory`.
```
Comment (olegrok) [inline CHANGELOG.md:25]: > (`number` in range [0, 1])

Let's add validation for this

File: cartridge/issues.lua [hunk 2]
```diff
@@ -147,9 +154,9 @@ local function list_on_instance()
                 slab_info.quota_used_ratio
             ),
         })
-    elseif items_used_ratio > vars.limits.fragmentation_threshold_warning
-    and arena_used_ratio > vars.limits.fragmentation_threshold_critical
-    and quota_used_ratio > vars.limits.fragmentation_threshold_critical
+    elseif items_used_ratio > limits.fragmentation_threshold_warning
+    and arena_used_ratio > limits.fragmentation_threshold_critical
+    and quota_used_ratio > limits.fragmentation_threshold_critical
```
Comment (olegrok) [inline cartridge/issues.lua:159]: > items_used_ratio > limits.**fragmentation_threshold_warning**
    and arena_used_ratio > **limits.fragmentation_threshold_critical**
    and quota_used_ratio > **limits.fragmentation_threshold_critical**

warning, critical, critical. Should it be "warning" in all places?

Comment (HustonMmmavr) [reply cartridge/issues.lua:159]: We decided to use thresholds as is in documentation

File: CHANGELOG.md (hunk 1 above)
Comment (olegrok) [reply CHANGELOG.md:25]: Also I think that TARANTOOL_FRAGMENTATION_THRESHOLD_WARNING should be less than TARANTOOL_FRAGMENTATION_THRESHOLD_CRITICAL

**Pull request merged**